django_project/cplus_api/api_views/user.py

```diff
@@ -8,6 +8,14 @@ class UserInfo(APIView):
     permission_classes = [IsAuthenticated]
 
     def get(self, request, *args, **kwargs):
+        user = request.user
         return Response(status=200, data={
```

Comment:
@zamuzakki can you add the swagger api schema for the response so we can also see what the API returns in the swagger.
![image](https://github.com/kartoza/cplus-api/assets/5819076/ed09290d-15eb-429c-80d8-8f8254846d35)